interpreter/dune

```diff
@@ -0,0 +1,29 @@
+(include_subdirs unqualified)
+
+(library
+ (name interpreter_lib)
+ (modules :standard \ main))
+
+(executable
+ (name main)
+ (modules main)
+ (libraries interpreter_lib)
+ (flags
+  (-open Interpreter_lib)))
+
+(subdir
+ text
+ (rule
+  (target lexer.ml)
+  (deps lexer.mll)
+  (action
+   (chdir
+    %{workspace_root}
+    (run %{bin:ocamllex} -ml -q -o %{target} %{deps}))))
+ (ocamlyacc
+  (modules parser)))
+
+(env
+ (dev
+  (flags
+   (-w -27-8))))
```

Comment:
Can we make this default to the same flags used in the Makefile?

Reply:
Good idea. Done.